packages/manager-components/src/components/table/table.spec.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { Table, TableProps } from './table.component';
+import { mockColumns, mocksData } from './table.stories';
+import { render, waitFor, within } from '../../utils/test.provider';
```

Comment:
it would make more sense to import directly from testing-library

Reply:
Sure it has been refactor in an other branch but I will delete this useless export and import